pypd/models/incident.py

```diff
@@ -110,3 +112,36 @@ def snooze(self, duration):
             api_key=self.api_key,
             data={'duration': duration},
         )
+
+    def merge(self, source_incidents, from_email=None,):
+        """Merge incidents into this incident."""
+        if from_email is None:
+            raise Exception('%s.merge requires \'from_email\' argument.')
+
+        add_headers = {'from': from_email, }
+
+        endpoint = '/'.join((self.endpoint, self.id, 'merge'))
+
+        transform_incident_to_id = lambda x: x['id'] if isinstance(x, Entity) else x
+        source_incidents_ids = list(map(transform_incident_to_id, source_incidents))
+
+        transform_incident_ids_to_api_object = lambda x: { 'type': 'incident_reference', 'id': x }
```

Comment:
nit: I'd prefer if the style was consistent for the `dict` here :) just without the extra whitespace